Web/src/AuthPage.js

```diff
@@ -47,7 +47,7 @@ export default function AuthPage() {
         await fetch("http://192.168.0.71:8080/" + endpoint, requestOptions).then(response => {
             response.json().then(data => {
                 console.log(data);
-                if (data.userUid != 'error') {
+                if (data.userUid !== 'error') {
```

Comment:
Is it ok ?

Reply:
Yes, it's checking if the values are the same AND if the type is the same. 
In JS, the condition 1 == '1' returns 'True'. 
While the condition 1 === '1' returns 'False'